packages/frontend-test-utils/src/app/createExtensionTester.tsx

```diff
@@ -245,34 +190,50 @@ export class ExtensionTester {
     return new ExtensionQuery(node);
   }
 
+  element(): JSX.Element {
+    const tree = this.#resolveTree();
+
+    const element = new ExtensionQuery(tree.root).data(
+      coreExtensionData.reactElement,
+    );
+
+    if (!element) {
+      throw new Error(
+        'No element found. Make sure the extension has a `coreExtensionData.reactElement` output, or use the `.get(myComponentDataRef)` method to get the component',
```

Comment:
```suggestion
        'No element found. Make sure the extension has a `coreExtensionData.reactElement` output, or use the `.get(...)` to access output data directly instead.
```

The `myComponentDataRef` seems a bit oddly specific here tbh, there are many different situations where `.element()` might not do what you want. Same thing goes for the error message below.

This also references `.get(...)`, which doesn't exist, it's `.data(...)`. I think we should just rename `data` -> `get` though tbh 😁